tests/tests_integration/test_create_folders.py

```diff
@@ -252,6 +252,9 @@ def test_datetime_flag_in_session(self, project):
         assert all([re.search(datetime_regexp, name) for name in ses_names])
         assert all([tags("time") not in name for name in ses_names])
 
+    def test_error_raised_with_skipped_session_input(self, project):
+        pass
```

Comment:
why create a test that does nothing? did you intende to fill it up later?

Reply:
good spot! In the end I don't think that test is worth doing. It might be addressed soon anyway if we allow skipping the session level.